Header header
Joints[] JointsArray
time[] t_k #desired execution of the trajectory
